client/src/components/Search/TagsSearch/TagsSearch.js

```diff
@@ -61,9 +61,10 @@ class TagsSearch extends React.Component {
           <div/>
         );
 
-        if (allowTagCreation && searchQuery !== '' && !selectedTags.find(tag => tag.name === searchQuery)
-            && !createdTags.find(tag => tag.name === searchQuery))
-        {
+        if (allowTagCreation && searchQuery !== ''
+            && !selectedTags.find(tag => tag.name.toLowerCase() === searchQuery.toLowerCase())
+            && !createdTags.find(tag => tag.name.toLowerCase() === searchQuery.toLowerCase())
+            && !visibleSearchResults.find(tag => tag.name.toLowerCase() === searchQuery.toLowerCase())) {
```

Comment:
This is good. But, say I input `stone` and there's no `Stone` or other casing. Would it be added as `stone` or do we auto-capitalize to `Stone`? Because I think we should capitalize

Reply:
Might also want to double check how this is handled for bulk